### ISSUE TITLE
fetal pleural effusion

### ISSUE THREAD

**Issue by sparkste** [obophenotype/human-phenotype-ontology#7002]
**Preferred term label:**
fetal pleural effusion

**Synonyms**

**Definition (free text, please give PubMed ID)**
fluid around the lungs in the fetal chest

**Parent term (use hpo.jax.org/app)**
pleural effusion

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Noonan syndrome and other RASopathies
Lymphatic disorders
Hematologic disorders
Inborn errors of metabolism
Viral infections
Many other single gene disorders

**Your nano-attribution (ORCID)**
0000-0002-8593-2186

**Comment by patt1morgan** [comment]
@rhiannonmellis, @EDempseyUK.   Good day!  This is another of the new term requests. When you have the time, could you please review and comment.  Thank you in advance for your help.

**Comment by pnrobinson** [comment]
added term
